torchaudio/transforms.py

```diff
@@ -549,17 +440,7 @@ def __call__(self, x_mu):
             x (FloatTensor or ndarray)
 
         """
-        mu = self.qc - 1.
-        if isinstance(x_mu, np.ndarray):
```

Comment:
Here is another instance of this conversion. Let's get rid of this kind of stuff in a future pull request.